ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -323,8 +326,19 @@ public static CCDCase getCCDCitizenCase(List<CCDCollectionElement<CCDAmountRow>>
             .interestClaimStartDate(LocalDate.now())
             .interestSpecificDailyAmount(BigDecimal.valueOf(10))
             .interestEndDateType(CCDInterestEndDateType.SUBMISSION)
+            .paymentStatus("success")
```

Comment:
this does not seem related to the PR and is not just a formatting change. This change should have been part of a separate PR.

Reply:
test coverage